### PR TITLE
Add Multi Platform Support 🐋 & some Documentation!

### PR DESCRIPTION
**Pull request by y-seq**
* Updated Documentation 
* Added generic Multi Platform Support 
* Implemented linxu/arm64 platform

### CODE REVIEW

File: base/BUILDING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+```shell
+docker build -t wibusystems/codemeter:8.00a --no-cache --build-arg CODEMETER_VERSION=8.0.5974.501 -f Dockerfile .
+```
+
+When Building mutliple platforms at once, buildx is recommended.  
+Example:
+
+```shell
+docker buildx build --platform linux/amd64,linux/arm64 -t wibusystems/cm_internal:8.10 -f Dockerfile .
```
Comment (wvoelkerde) [inline base/BUILDING.md:37]: Ist das Absicht mit "internal"?: wibusystems/cm_internal

File: base/BUILDING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+### Additonal Build Parameters
+
+When cross building images for another architecture, the additonal `platform` parameter may be passed when building. Currently only amd64 and arm64 CPU Architectures are supported.  
+Example:
+
+```shell
+docker build -t wibusystems/codemeter --no-cache --platform=linux/arm64 -f Dockerfile .
+```
+
+Changing the CodeMeter Version can be done by placing another CodeMeter-Deb Package into this directory and passing the build argument `CODEMETER_VERSION`.
```
Comment (jo-wibu) [inline base/BUILDING.md:26]: -> version (kleines v)

File: base/BUILDING.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# How to Build  
+
+This is a short description on how the images are built
+
+## Context Preparation
+
+The `Dockerfile` assumes that the deb package for this arch is located next at the build context root (usually next to the docker file). Therefore it may be required to copy that file into your build context.
```
[the remark this answers is not in the document]
Comment (y-seq) [reply base/BUILDING.md]: Updated in 3aeea75

File: base/BUILDING.md (hunk 1 above)
Comment (y-seq) [reply base/BUILDING.md]: No thats a typo :(

**Review by wvoelkerde — APPROVED** [review]
ok

**Review by jo-wibu — APPROVED** [review]

**Pull request merged**